quidel_covidtest/delphi_quidel_covidtest/backfill.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""Store backfill data."""
+import os
+import glob
+from datetime import datetime
+
+import pandas as pd
+
+from delphi_utils import GeoMapper
+
+
+gmpr = GeoMapper()
+
+def store_backfill_file(df, _end_date, backfill_dir):
+    """
+    Store county level backfill data into backfill_dir.
+
+    Parameter:
+        df: pd.DataFrame
+            Pre-process file at ZipCode level
+        _end_date: datetime
+            The most recent date when the raw data is received
+        backfill_dir: str
+            specified path to store backfill files.
+    """
+    backfilldata = df.copy()
+    backfilldata = gmpr.replace_geocode(backfilldata, from_code="zip", new_code="fips",
```

Comment:
You are right. Yeah, it's easier to do that using Python

Reply:
I add a `state_id` column using `GeoMapper` so that we are able to easily at state-level data in the backfill correction pipeline and avoid storing extra data.